tests/worksheets/src/test/scala/tests/worksheets/WorksheetSuite.scala

```diff
@@ -382,6 +382,24 @@ class WorksheetSuite extends BaseSuite {
        |""".stripMargin
   )
 
+  checkExportableEvaluations(
+    "simple",
+    "val x = 3",
+    ju.Optional.of("val x = 3 // : Int = 3")
+  )
+
+  checkExportableEvaluations(
+    "continuation",
+    """val x = println("badabing\nbadaboom")""",
+    ju.Optional.of("""val x = println("badabing\nbadaboom") // badabing…""")
```

Comment:
Should we try to add lines below in that case and continue the output?

Reply:
So I've gone back and forth with this, and I really don't know what's best because if with a simple example it gets a bit verbose. For example, the following:

```scala
case class LongThing(a: String, b: String, c: String, d: String)

val longThing1 = LongThing("one", "two", "three", "four")
val longThing2 = LongThing("one", "two", "three", "four")
val longThing3 = LongThing("one", "two", "three", "four")
val longThing4 = LongThing("one", "two", "three", "four")
val longThing5 = LongThing("one", "two", "three", "four")

val myListOfThings =
  List(longThing1, longThing2, longThing3, longThing4, longThing5)

myListOfThings.map(_.copy(c = "now I'm five"))
```
With the current behavior, if I copy this out it looks like this:

```scala
case class LongThing(a: String, b: String, c: String, d: String)

val longThing1 = LongThing("one", "two", "three", "four") // : LongThing = LongThing("one", "two", "three", "four")
val longThing2 = LongThing("one", "two", "three", "four") // : LongThing = LongThing("one", "two", "three", "four")
val longThing3 = LongThing("one", "two", "three", "four") // : LongThing = LongThing("one", "two", "three", "four")
val longThing4 = LongThing("one", "two", "three", "four") // : LongThing = LongThing("one", "two", "three", "four")
val longThing5 = LongThing("one", "two", "three", "four") // : LongThing = LongThing("one", "two", "three", "four")

val myListOfThings =
  List(longThing1, longThing2, longThing3, longThing4, longThing5) // : List[LongThing] = List(LongThing("one","two","three"…

myListOfThings.map(_.copy(c = "now I'm five")) // : List[LongThing] = List(LongThing("one", "two", "now I'm five", "four"),L…
```
However, I'm assuming that if I included everything it'd be like:

```scala
case class LongThing(a: String, b: String, c: String, d: String)

val longThing1 = LongThing("one", "two", "three", "four") // : LongThing = LongThing("one", "two", "three", "four")
val longThing2 = LongThing("one", "two", "three", "four") // : LongThing = LongThing("one", "two", "three", "four")
val longThing3 = LongThing("one", "two", "three", "four") // : LongThing = LongThing("one", "two", "three", "four")
val longThing4 = LongThing("one", "two", "three", "four") // : LongThing = LongThing("one", "two", "three", "four")
val longThing5 = LongThing("one", "two", "three", "four") // : LongThing = LongThing("one", "two", "three", "four")

val myListOfThings =
  List(longThing1, longThing2, longThing3, longThing4, longThing5)
// myListOfThings: List[LongThing] = List(
//   LongThing("one", "two", "three", "four"),
//   LongThing("one", "two", "three", "four"),
//   LongThing("one", "two", "three", "four"),
//   LongThing("one", "two", "three", "four"),
//   LongThing("one", "two", "three", "four")
// )

myListOfThings.map(_.copy(c = "now I'm five"))
// res0: List[LongThing] = List(
//   LongThing("one", "two", "now I'm five", "four"),
//   LongThing("one", "two", "now I'm five", "four"),
//   LongThing("one", "two", "now I'm five", "four"),
//   LongThing("one", "two", "now I'm five", "four"),
//   LongThing("one", "two", "now I'm five", "four")
// )
```
I know for me personally there are a lot of times in worksheets I have _very_ long output. I'm really unsure. What do you think?

Reply:
We could adjust it using a setting? There might be situations that people want to get the full output, but will be unable to do so.

Reply:
Yea, the more I think about it, the more it makes sense to have it all there.